boilerplate/app/i18n/hi.ts

```diff
@@ -0,0 +1,131 @@
+import demoHi from "./demo-hi"
+import { Translations } from "./en"
+
+const hi: Translations = {
+  common: {
+    ok: "ठीक है!",
+    cancel: "रद्द करें",
+    back: "वापस",
+    logOut: "लॉग आउट", // @demo remove-current-line
+  },
+  welcomeScreen: {
+    postscript:
+      "पीएसटी - शायद आपका ऐप ऐसा नहीं दिखता है। (जब तक कि आपके डिजाइनर ने आपको ये स्क्रीन नहीं दी हों, और उस स्थिति में, इसे लॉन्च करें!)",
+    readyForLaunch: "आपका ऐप, लगभग लॉन्च के लिए तैयार है!",
+    exciting: "(ओह, यह रोमांचक है!)",
+    letsGo: "चलो चलते हैं!", // @demo remove-current-line
+  },
+  errorScreen: {
+    title: "कुछ गलत हो गया!",
+    friendlySubtitle:
+      "यह वह स्क्रीन है जो आपके उपयोगकर्ता प्रोडक्शन में देखेंगे जब कोई त्रुटि होगी। आप इस संदेश को कस्टमाइज़ करना चाहेंगे (जो `app/i18n/hi.ts` में स्थित है) और शायद लेआउट भी (`app/screens/ErrorScreen`). यदि आप इसे पूरी तरह से हटाना चाहते हैं, तो `app/app.tsx` में <ErrorBoundary> कंपोनेंट की जांच करें।",
+    reset: "ऐप रीसेट करें",
+    traceTitle: "%{name} स्टैक से त्रुटि", // @demo remove-current-line
+  },
+  emptyStateComponent: {
+    generic: {
+      heading: "इतना खाली... इतना उदास",
+      content:
+        "अभी तक कोई डेटा नहीं मिला। रीफ्रेश करने या ऐप को पुनः लोड करने के लिए बटन पर क्लिक करें।",
+      button: "चलो फिर से कोशिश करते हैं",
+    },
+  },
+  // @demo remove-block-start
+  errors: {
+    invalidEmail: "अमान्य ईमेल पता।",
+  },
+  loginScreen: {
+    logIn: "लॉग इन करें",
```

Comment:
Pravesh karein is a more accurate translation imo.

Reply:
"प्रवेश करें" (Pravesh karein) means "enter" or "access." It's a valid translation but it might not be the most accurate translation for "Log In".

"लॉग इन करें" (Log in karein) is a transliteration and is commonly understood, especially in tech contexts, since it directly references the English term "Log in."

Reply:
![Screenshot 2024-09-01 at 6 09 56 PM](https://github.com/user-attachments/assets/23e5c588-6284-4c8a-9600-295d7f160c74)
When I wrote that comment this is what I was referring to. This is facebook's Tamil login page. They're translated "login" to "uL nuLaiga" which is the same as pravesh karein but weirdly, for hindi, it is "login karein"
![Screenshot 2024-09-01 at 6 11 28 PM](https://github.com/user-attachments/assets/8a97168b-6dc8-44a9-a5d0-3a2624be67a5)

Reply:
I'm alright with login karein.